box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxRecentItem.java

```diff
@@ -0,0 +1,80 @@
+package com.box.androidsdk.content.models;
+
+import com.eclipsesource.json.JsonObject;
+
+import java.util.Date;
+
+/**
+ * A class representing Box recent item
+ */
+public class BoxRecentItem extends BoxJsonObject {
+    private static final long serialVersionUID = -2642748896882484887L;
+
+    public static final String TYPE = "recent_item";
+
+    public static final String FIELD_INTERACTION_TYPE = "interaction_type";
```

Comment:
I found a model "BoxDownload", which I used as reference. It has private accessibility for static properties, so I believe it should be fine to narrow it down.